### ISSUE TITLE
cannot resolve the account name "@krystal.swor.ds" when composing posts

### ISSUE THREAD

**Issue by deceptionagent** [bluesky-social/social-app#2517]
when composing a post, typing "@krystal.swor.ds" does not find the account in the live post composer or result in resolved account reference after posting. attached screenshots show that this is a valid account name and that i can otherwise resolve account names of other people.

i wasn't sure if this was proto or client, so i went client.

platform: iOS, mobile web (safari), desktop web

i do not have an android device on which to test.

![image](https://github.com/bluesky-social/social-app/assets/29762578/e425ba8e-ef6e-46fe-90bb-090ffe6260ca)
![image](https://github.com/bluesky-social/social-app/assets/29762578/7b627acf-dfc6-48ea-960e-466d6455539e)
![image](https://github.com/bluesky-social/social-app/assets/29762578/0c5a6787-8c71-4963-a813-578c98c12406)

**Comment by wtlgo** [comment]
This might happen because they didn't confirm their email address. I noticed that this happens to new users I invite, and after they confirm their emails it becomes normal.

**Comment by Bossett** [comment]
https://bsky.app/profile/bossett.bsky.social/post/3kispucxmkf2q - tagging the old handle works, even though it was unable to resolve either handle via auto-complete in the composer.

**Comment by pfrazee** [comment]
Oh yeah, we don't support Handshake TLDs. This isn't strictly enforced per-se; it's just that none of the DNS resolution architecture uses it.

**Comment by deceptionagent** [comment]
yeah, now that i know tbis is handshake-related i would agree it ain't a thing you should support. would you like to close, or keep this for disallowing them?
